CHANGELOG.md

```diff
@@ -59,6 +59,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * (modules/core/04-channel) [\#1279](https://github.com/cosmos/ibc-go/pull/1279) Add selected channel version to MsgChanOpenInitResponse and MsgChanOpenTryResponse. Emit channel version during OpenInit/OpenTry
 * (app/29-fee) [\#1305](https://github.com/cosmos/ibc-go/pull/1305) Change version string for fee module to `ics29-1`
 * (app/29-fee) [\#1341](https://github.com/cosmos/ibc-go/pull/1341) Check if the fee module is locked and if the fee module is enabled before refunding all fees
+* (modules/core/04-channel) [\#1317](https://github.com/cosmos/ibc-go/pull/1317)  Remove crossing hellos from ChanOpenTry
```

Comment:
We will need to add migration docs as well so application developers know this has been changed